examples/pylintrc

```diff
@@ -354,13 +354,20 @@ min-similarity-lines=4
 
 
 [SPELLING]
+# To enable spellchecking, install hunspell and its python bindings
+# Hunspell's python bindings are available on PyPI as the "hunspell" package
 
-# Limits count of emitted suggestions for spelling mistakes
-max-spelling-suggestions=4
+# Spelling dictionary name (e.g. "en_US")
+spelling-dict-name=
 
-# Spelling dictionary name. Available dictionaries: none. To make it working
-# install python-enchant package.
-spelling-dict=
+# Paths string of directories where Hunspell dictionaries are installed.
+# To get a list of available paths, run `hunspell -D` in a shell and look for
+# the AVAILABLE DICTIONARIES section.
+# Pylint will search under these directories for the following files:
+#   <spelling-dict-name>.dic
```

Comment:
Is this something that can also be used on Windows for instance? while currently `pyenchant` is unmaintained, we still offer Windows support as long as the library is installable on that platform.